supertokens_python/recipe/emailpassword/recipe.py

```diff
@@ -49,22 +51,31 @@
 class EmailPasswordRecipe(RecipeModule):
     recipe_id = 'emailpassword'
     __instance = None
+    email_delivery: Union[EmailDeliveryIngredient[TypeEmailPasswordEmailDeliveryInput], None]
 
     def __init__(self, recipe_id: str, app_info: AppInfo,
                  sign_up_feature: Union[InputSignUpFeature, None] = None,
                  reset_password_using_token_feature: Union[InputResetPasswordUsingTokenFeature, None] = None,
                  email_verification_feature: Union[InputEmailVerificationConfig, None] = None,
                  override: Union[InputOverrideConfig, None] = None,
```

Comment:
This should also take `email_delivery: Union[EmailDeliveryConfig< TypeEmailPasswordEmailDeliveryInput>, None] = None` too.

Reply:
Fixed